hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -135,7 +135,7 @@ def test_get_calibration_from(self):
 class Test_quantification:
 
     def setUp(self):
-        s = EDSTEMSpectrum(np.ones([2, 1024]))
+        s = EDSTEMSpectrum(np.ones([2, 2, 1024]))
         energy_axis = s.axes_manager.signal_axes[0]
```

Comment:
Is there a reason why you've made this a 'spectrum image' i.e. 2 navigation dimensions, rather than a line scan as it was before? As far as I can see all you've ended up doing is doubling the number of numbers to be compared in the tests, but haven't actually added any functionality to the tests?